pygotham/admin/__init__.py

```diff
@@ -3,12 +3,16 @@
 import importlib
 import pkgutil
 
+import arrow
+from flask import g
 from flask_admin import Admin
 from flask_admin.base import AdminIndexView, MenuLink
 from flask.ext.admin.contrib.sqla import ModelView
 from flask_login import current_user
+from sqlalchemy import or_
 
 from pygotham import factory, filters
+from pygotham.events.models import Event
```

Comment:
Not strictly, no. I'll update this to remove it for consistency with other parts of the code.